uint8 SPHERE=1
uint8 BOX=2
uint8 CONE=3
uint8 CYLINDER=4
uint8 OPEN_CYLINDER=5
uint8 CAPSULE=6
uint8 TRIANGLE=7
uint8 QUAD=8
uint8 LINES=9
uint8 LINESTRIP=10
uint8 MESH=11

string system_name
string rigid_body_name
string material_name
uint8 geometry_type
float64[] geometry_params
float64[] color_rgba
